docs/main.css

```diff
@@ -0,0 +1,326 @@
+@import url("../utils/reset.css");
+@import url("../utils/color.css");
+@import url("../utils/utility.css");
+/* genral styles for documentation page */
+:root {
+  --font-poppins: "Poppins", sans-serif;
+  --font-montserrat: "Montserrat", sans-serif;
+  --cta-bgcolor: var(--DARK-BASE);
+  --cta-fcolor: var(--NEUTRAL);
+  --heading--color: var(--DARK-BASE);
+  --author-name-color: var(--DARK-BASE);
+  --azure: azure;
+}
+
+/*setting default font-size to 16px*/
+html {
+  font-size: 62.5%;
+  font-family: var(--font-montserrat);
+}
+
+
+body {
+  background: var(--azure);
+  font-size: 1.6rem;
+}
+
+/* Layout */
+
+.container {
+  max-width: 1440px;
```

Comment:
use rem here

Reply:
Okay! thanks for pointing it out 👍
.container class is very important in this project :D

Reply:
Hey I just changed the units to rem
please approve this changes